NitroxServer-Subnautica/Program.cs

```diff
@@ -183,8 +183,9 @@ private static Assembly CurrentDomainOnAssemblyResolve(object sender, ResolveEve
             }
 
             // Load DLLs where this program (exe) is located
-            string dllPath = Path.Combine(Path.GetDirectoryName(Assembly.GetEntryAssembly().Location), "lib", dllFileName);
-            if (!File.Exists(dllPath))
+            string dllPath = Path.Combine(Path.GetDirectoryName(Assembly.GetEntryAssembly()?.Location) ?? "", "lib", dllFileName);
+            // Prefer to use Newtonsoft dll from game instead of our own due to protobuf issues. TODO: Remove when we do our own deserialization of game data instead of using the game's protobuf.
+            if (dllPath.IndexOf("Newtonsoft.Json.dll", StringComparison.OrdinalIgnoreCase) >= 0 || !File.Exists(dllPath))
```

Comment:
Is `IndexOf` better then `Contains`?

Reply:
Contains doesn't have an ignore case flag. Either way would technically work though. As long as the case matches exactly.